deployment/CP04OSSM_Deploy.csv

```diff
@@ -1,342 +1,373 @@
+AR48,,,,CP04OSSM-MFD35-05-PCO2WB000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-PCO2WB-C0097,39,-70,,,,
+AR48,,,,CP04OSSM-MFD35-06-PHSEND000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-PHSEND-P0096,39,-70,,,,
+AR48,,,,CP04OSSM-MFD37-00-DCLENG000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGCON-MDCL37-00013,39,-70,,,,
+AR48,,,,CP04OSSM-MFD37-01-OPTAAD000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-OPTAAD-00222,39,-70,,,,
+AR48,,,,CP04OSSM-MFD37-03-CTDBPE000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-CTDBPE-50004,39,-70,,,,
+AR48,,,,CP04OSSM-MFD37-04-DOSTAD000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-DOSTAD-00392,39,-70,,,,
+AR48,,,,CP04OSSM-MFD37-07-ZPLSCC000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-ZPLSCC-55077,39,-70,,,,
+AR48,,,,CP04OSSM-RIC21-00-CPMENG000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGCON-NCPM02-00009,39,-70,,7,,
+AR48,,,,CP04OSSM-RID26-00-DCLENG000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGCON-NDCL26-00015,39,-70,,7,,
+AR48,,,,CP04OSSM-RID26-04-VELPTA000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-VELPTA-12229,39,-70,,7,,
+AR48,,,,CP04OSSM-RID26-06-PHSEND000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-PHSEND-P0099,39,-70,,7,,
+AR48,,,,CP04OSSM-RID26-07-NUTNRB000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-NUTNRB-01107,39,-70,,7,,
+AR48,,,,CP04OSSM-RID26-08-SPKIRB000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-SPKIRB-00289,39,-70,,7,,
+AR48,,,,CP04OSSM-RID27-00-DCLENG000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGCON-NDCL27-00009,39,-70,,7,,
+AR48,,,,CP04OSSM-RID27-01-OPTAAD000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-OPTAAD-00206,39,-70,,7,,
+AR48,,,,CP04OSSM-RID27-02-FLORTD000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-FLORTD-01214,39,-70,,7,,
+AR48,,,,CP04OSSM-RID27-03-CTDBPC000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-CTDBPC-50003,39,-70,,7,,
+AR48,,,,CP04OSSM-RID27-04-DOSTAD000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-DOSTAD-00391,39,-70,,7,,
+AR48,,,,CP04OSSM-SBC11-00-CPMENG000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGCON-BCPM01-50002,39,-70,,0,,
+AR48,,,,CP04OSSM-SBD11-00-DCLENG000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGCON-BDCL11-50002,39,-70,,0,,
+AR48,,,,CP04OSSM-SBD11-01-MOPAK0000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-MOPAK0-44005,39,-70,,0,,
+AR48,,,,CP04OSSM-SBD11-02-HYDGN0000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-HYDGN0-46156,39,-70,,0,,
+AR48,,,,CP04OSSM-SBD11-06-METBKA000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-METLGR-00038,39,-70,,0,,
+AR48,,,,CP04OSSM-SBD12-00-DCLENG000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGCON-BDCL12-50002,39,-70,,0,,
+AR48,,,,CP04OSSM-SBD12-03-HYDGN0000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-HYDGN0-41552,39,-70,,0,,
+AR48,,,,CP04OSSM-SBD12-04-PCO2AA000,12,1,2020-10-28T00:00:00,,CGMCP-04OSSM-00012,,CGINS-PCO2AA-35260,39,-70,,0,,
```

Comment:
Looks good. Reviewed CUIDs and compared SNs with config sheet

Reply:
Looks good.